lib/widgets/matrix.dart

```diff
@@ -94,8 +94,16 @@ class MatrixState extends State<Matrix>
     return tomServerUrlInterceptor.baseUrl != null;
   }
 
+  bool get canDeleteToMDatabase {
+    final lastClientTwakeSupport =
+        widget.clients.where((client) => client.homeserver != null).length == 1;
```

Comment:
still not understand this condition

Reply:
- please prevent delete all `tom-db` when client logout, only do it when no client available.
- try to delete specific client in `tom-db` when this client logout.